bin/import_ru.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
```

Comment:
nit: I would not be against something in this that states that this is only fitted for running CI on TaskCluster, to prevent people from thinking anything else

Reply:
added